src/index.html

```diff
@@ -6,8 +6,41 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+
+    <ul class="tree">
+      <li>Fruit</li>
+      <ul>
+        <li>Red</li>
+        <ul>
+          <li>Cherry</li>
+          <li>Strawberry</li>
+          <li>Watermelon</li>
+          <li>Pomegranate</li>
+        </ul>
+        <li>Yellow</li>
+        <ul>
+          <li>Banana</li>
+          <li>Pineapple</li>
+          <li>Mango</li>
+        </ul>
+      </ul>
+      <li>Fish</li>
+      <ul>
+        <li>Ocean fishes</li>
+        <ul>
+          <li>Blue Marlin</li>
+          <li>Sailfish</li>
+          <li>Dorado</li>
+        </ul>
+        <li>Freshwater fish</li>
+        <ul>
+          <li>Oscar</li>
+          <li>Mollies</li>
+          <li>Goldfish</li>
+        </ul>
+      </ul>
+    </ul>
+
     <script type="text/javascript" src="scripts/main.js"></script>
```

Comment:
why do we put the `script` at the end of the `body` instead of `head`???

Reply:
moved it to the `head`